spec/models/manageiq/providers/vmware/cloud_manager_spec.rb

```diff
@@ -120,6 +120,10 @@
 
         @ems.vm_create_snapshot(vm, snapshot_options)
       end
+
+      it 'supports snapshot create' do
+        expect(vm.supports_snapshot_create?).to be_truthy
+      end
     end
```

Comment:
Please add  another test case where you first manually add one snapshot to the `vm` prior checking for `supports_snapshot_create?`. Just to make sure your `supports` did override the one from core.